File: app/src/main/java/com/honey96dev/homemadeproduct/CustomerSectionsPagerAdapter.java
package com.honey96dev.homemadeproduct;


import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;
import android.util.Log;

import java.util.ArrayList;

/**
 * A {@link FragmentPagerAdapter} that returns a fragment corresponding to
 * one of the sections/tabs/pages.
 */
public class CustomerSectionsPagerAdapter extends FragmentPagerAdapter {

    ArrayList<Fragment> sections;

    public CustomerSectionsPagerAdapter(FragmentManager fm) {
        super(fm);
        sections = new ArrayList<>();
        sections.add(CustomerProductListFragment.newInstance());
        sections.add(CustomerCartListFragment.newInstance());
    }

    @Override
    public Fragment getItem(int position) {
        Log.e("position", String.valueOf(position));
        return sections.get(position);
    }

    @Override
    public int getCount() {
        return sections.size();
    }
}

File: app/src/main/java/com/honey96dev/homemadeproduct/p4customer/CustomerSectionsPagerAdapter.kt
package com.honey96dev.homemadeproduct.p4customer


import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter
import android.util.Log
import com.honey96dev.homemadeproduct.p4manager.CustomerCartListFragment
import com.honey96dev.homemadeproduct.p4manager.CustomerStoreListFragment

import java.util.ArrayList

/**
 * A [FragmentPagerAdapter] that returns a fragment corresponding to
 * one of the sections/tabs/pages.
 */
class CustomerSectionsPagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    internal var sections: ArrayList<Fragment>

    init {
        sections = ArrayList()
        sections.add(CustomerStoreListFragment.newInstance())
        sections.add(CustomerCartListFragment.newInstance())
    }

    override fun getItem(position: Int): Fragment {
        Log.e("position", position.toString())
        return sections[position]
    }

    override fun getCount(): Int {
        return sections.size
    }
}
